activemodel/lib/active_model/errors.rb

```diff
@@ -112,6 +112,17 @@ def merge!(other)
       @details.merge!(other.details) { |_, ary1, ary2| ary1 + ary2 }
     end
 
+    # Removes the given keys from the errors.
+    #
+    #   person.errors.keys                  # => [:name, :age, :gender]
+    #   person.errors.slice!(:age, :gender) # => { :age=>["cannot be nil"], :gender=>["cannot be nil"]}
+    #   person.errors.keys(:name, :age)     # => [:name]
+    def slice!(*keys)
+      keys.inject({}) do |acc, key|
```

Comment:
The `messages` and `details` are both hashes. I think it makes more sense for us to just use `slice!` on those like we do with `merge!`, right?